.travis.yml

```diff
@@ -0,0 +1,39 @@
+language:
+  - cpp
+
+sudo: false
+
+addons:
+  apt:
+    packages:
+    - g++ make
+    - libsdl1.2-dev
+    - libjpeg62-turbo-dev
+    - libmpeg2-4-dev
+    - libogg-dev
+    - libvorbis-dev
+    - libflac-dev
+    - libmad0-dev
+    - libpng-dev
+    - libtheora-dev
+    - libfaad-dev
+    - libfluidsynth-dev
+    - libfreetype6-dev
+    - zlib1g-dev
+
+branches:
+ only:
+   - master
+
+compiler:
+  - gcc
+  - clang
+
+os:
+  - linux
+
+script:
+  - ./configure --enable-all-engines
+  - make
+  - make test
```

Comment:
Is this a good idea? With the given configure set up our tests will always fail (to link due to EventRecorder cross dependencies). Maybe we should pass `--disable-eventrecorder` to `configure` in this setup.